config/sets/symfony/symfony53.php

```diff
@@ -91,6 +96,21 @@
         ),
     ]);
 
+    $rectorConfig->ruleWithConfiguration(AddParamTypeDeclarationRector::class, [
+        new AddParamTypeDeclaration(
+            'Symfony\Component\Form\DataMapperInterface',
+            'mapFormsToData',
+            0,
+            new ObjectType(Traversable::class)
+        ),
+        new AddParamTypeDeclaration(
+            'Symfony\Component\Form\DataMapperInterface',
+            'mapDataToForms',
+            1,
+            new ObjectType(Traversable::class)
+        ),
+    ]);
```

Comment:
Hi, thanks for the fix. Could you add a comment with this feature here?

Reply:
done!

Reply:
Thanks :+1: